### ISSUE TITLE
Link to .env not working with Remote SSH

### ISSUE THREAD

**Issue by tszshingt** [rangav/thunder-client-support#821]
**Describe the bug**
When I click "Choose file" with the "Link to .env file" option selected, it opens up the dialog box to choose file on my local computer, instead of file in the remote computer.

**To Reproduce**
1. Connect to a remote computer using Remote SSH extension
2. Add a new environment in Thunder Client.
3. Check the "Link to .env file" option and click "Choose file". It opens up the dialog box in my local computer.

**Expected behavior**
It should allow me to choose file from the remote computer.

**Platform:** 
 - OS: MacOS Monterey 12.6 (local); Ubuntu 18.04.6 (remote)
 - vscode version: 1.71.2
 - extension version: 1.19.7

**Solution:**
Open the dialog box to choose file from the remote computer, similar to how import/export is done; or allow us to specify a full file path as an input string.

Thank you!

**Comment by rangav** [comment]
Hi @tszshingt you need to add vscode setting 

try adding this as mentioned in comment 
https://github.com/rangav/thunder-client-support/issues/158#issuecomment-851058322

**Comment by tszshingt** [comment]
Hi @rangav , I just tried add the following to my settings file, but I still have the same issue. The import/export of collection/environment files works correctly though (read/write in the remote computer). Only the "Choose file" button under "Link to .env file" option is giving me issue.
```
  "remote.extensionKind": {
    "rangav.vscode-thunder-client": ["workspace"]
  }
```
On another note, with the following setting, importing collection/environment file opens up dialog to choose file in the remote computer, but exporting collection/environment file goes to my local computer. The "Choose file" button under "Link to .env file" option still goes to my local computer. So, I know this setting is having an effect.
```
  "remote.extensionKind": {
    "rangav.vscode-thunder-client": ["ui"]
  }
```

Also, I could export the environment file, update the `envFile` key in the JSON to point to a file in the remote computer, re-import it, and it will work. But I want to be able to "Link to .env file" in the Global or Local Environment which I cannot currently import/export.

Thank you for looking into this! :)

**Comment by rangav** [comment]
related #648

**Comment by rangav** [comment]
This bug is now fixed and published to marketplace, please update to `v1.20.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v1.20.0

Please let me know your feedback.
